Tasks/JenkinsQueueJob/jobsearch.ts

```diff
@@ -119,7 +119,11 @@ export class JobSearch {
             for (var i in causes) {
                 var job = thisSearch.queue.findJob(causes[i].upstreamUrl, causes[i].upstreamBuild);
                 if (job) { // we know about it
-                    if (job.state == JobState.Streaming || job.state == JobState.Finishing || job.state == JobState.Done) {
+                    if (job.state == JobState.Streaming ||
+                        job.state == JobState.Finishing ||
+                        job.state == JobState.Downloading ||
+                        job.state == JobState.Queued ||
```

Comment:
How does this work in a pipelined scenario?  If we queued the job, and don't wait for it to complete, how are we capturing the downstream pipelined jobs?